lib/sink.js

```diff
@@ -159,6 +160,63 @@ module.exports = class SinkGCS extends EventEmitter {
         return !!(result && result[0]);
     }
 
+    _getPrefixes(directoryName) {
+        let prefix;
+        let filterPrefix;
+        if (directoryName === '/' || !directoryName) {
+            filterPrefix = '.';
+        } else {
+            prefix = `${directoryName.replace(/^\//, '').replace(/\/$/, '')}/`;
+            filterPrefix = prefix;
+        }
+        return { filterPrefix, prefix };
+    }
+
+    _getPrefixFilterIterator(filterPrefix) {
```

Comment:
Symbols for privacy?

Reply:
Sure!

Reply:
None of them touch `this`, so I'd just lift them out of the `class` itself

Reply:
Sure @SimenB. Will add symbol to `_getGcsFile`.

Reply:
done, moved 2 fucntions to scope - and made one method private.